locales/messages.de.json

```diff
@@ -1,35 +1,35 @@
 {
   "AddPortSucceded": "Der Datei „vcpkg.json“ konnten erfolgreich Ports hinzugefügt werden.",
-  "AddTripletExpressionNotAllowed": "Fehler: Tripelausdrücke sind hier nicht zulässig. Erwägen Sie, „{package_name}:{triplet}“ in „{package_name}“ zu ändern.",
+  "AddTripletExpressionNotAllowed": "Fehler: Triplettausdrücke sind hier nicht zulässig. Erwägen Sie, „{package_name}:{triplet}“ in „{package_name}“ zu ändern.",
   "AllFormatArgsRawArgument": "Formatzeichenfolge „{value}“ enthält ein nicht formatiertes Formatargument.",
   "AllFormatArgsUnbalancedBraces": "nicht ausgeglichene Klammer in der Formatzeichenfolge „{value}“",
   "AwsAttemptingToFetchPackages": "Es wird versucht, {count} Pakete aus AWS abzurufen.",
-  "AwsFailedToDownload": "AWS konnte nicht herunterladen, Exitcode: {exit_code}",
-  "AwsRestoredPackages": "{count} Pakete von AWS-Servern wurden in {elapsed} wiederhergestellt.",
-  "AwsUploadedPackages": "Binärdateien wurden auf {count} AWS-Server hochgeladen",
```

Comment:
This is/was logically wrong.

Reply:
I don't think it is; it's just a weird message, but that's an actual translation of the original english "Uploaded packages to {count} AWS servers"